tests/test_serializers.py

```diff
@@ -31,6 +37,7 @@ class Meta:
 class UserSerializer(ModelSerializer):
     class Meta:
         model = User
+        fields = '__all__'
```

Comment:
`fields` could only be a list or tuple until DRF 3.3.0.  Maybe we should bump the requirement to that version?